code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -656,6 +656,11 @@
 					user.put_in_inactive_hand(cut_scalp) //Put it in the user's offhand if possible.
 					victim.h_style = "Bald"
 					victim.update_hair() //tear the hair off with the scalp
+					if(user.hunter_data.prey == target)
+						to_chat(src, SPAN_YAUTJABOLD("You have claimed the scalp of [target] as your trophy."))
+						user.emote("roar2")
+						message_all_yautja("[user.real_name] has claimed the scalp of [target] as their trophy.")
+						user.hunter_data.prey = null
```

Comment:
this should be where they're made permanently dead in line with the butcher verb, no?

Reply:
The only additional "mechanic" is the fact that the scalp now counts as a trophy and completes your hunt. I didnt add any permakill that wasnt there due to it not being the main intent.

If there **should** be a permakill in this proc, I can always add it if desired.